packages/python-runner/runner.py

```diff
@@ -205,7 +205,9 @@ async def run_instance(self, config, args):
             self.logger.info(f'Sending PANG with {consumes}')
             send_encoded_msg(monitoring, msg_codes.PANG, consumes)
 
-        if asyncio.iscoroutine(result):
+        if isinstance(result, types.AsyncGeneratorType):
+            result = Stream.read_from(result)
```

Comment:
This is basically very similar to [what we have in JS runner](https://github.com/scramjetorg/transform-hub/blob/devel/packages/runner/src/runner.ts#L476-L480) so it should align runner behaviors :+1: